DependencyInjection/App/Presenters/AnotherPresenter.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
+
+namespace App\Presenters;
+
+use App;
+use Vendor;
+
+class AnotherPresenter extends Vendor\Presenters\SomePresenter
+{
+	/**
+	 * @var \Vendor\Services\AnotherVendorService
```

Comment:
Pokud mám být striktní, tak je naimportovaný vendor NS, takže ne FQN. Myslím, že v CS je to taky takto uvedeno.